nixos/modules/programs/steam.nix

```diff
@@ -0,0 +1,27 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.steam;
+in {
+  options.programs.steam = {
+    enable = mkEnableOption "steam";
+  };
+
+  config = mkIf cfg.enable {
+    hardware.opengl = { # glXchooseVisual failed https://github.com/NixOS/nixpkgs/issues/47932
```

Comment:
the link could use a little bit more context. After reading it I'm still not sure whats the connection here